includes/sensor_interface.h

```diff
@@ -0,0 +1,36 @@
+#ifndef SENSOR_INTERFACE_H
+#define SENSOR_INTERFACE_H
+
+#include <structs.h>
+
+extern struct k_lifo sensor_value_lifo;
+
+
+/* This function gets the device from the device tree. Note that the device
+ * should have "bosch,bme280" as compatible string.
+ * 
+ * @param None
+ * @return Pointer to the device, or NULL if the device is not found.
+ */
+const struct device *get_bme280_device(void);
```

Comment:
I agree if the interface occupies more then one sensor/sub-interface. I think that an abstraction is necessary when there will be changes in the implementation in future. On a project, works with only BME280 and uses abstraction for BME280 itself, it would not be a good choice to add extra-layer.